06-Resources-For-Learning-R.Rmd

```diff
@@ -5,15 +5,26 @@ date: "17/03/2021"
 output: html_document
 ---
 
-# Resources for learning R {#learnR}
+# Resources for learning R {#learnR} and Git
 
 ## Books
 
-* [An Introduction to R](https://intro2r.com/index.html)
-* [R for Health Data Science](https://argoshare.is.ed.ac.uk/healthyr_book/)
+* [Alex Douglas, Deon Roos, Francesca Mancini, Ana Couto & David Lusseau. An Introduction to R. Github Bookdown, 2021](https://intro2r.com/index.html)
+* [Ewen Harrison, Riinu Pius. R for Health Data Science. Chapman and Hall/CRC, 2021](https://argoshare.is.ed.ac.uk/healthyr_book/)
+* [Garrett Grolemund and Hadley Wickham. R for Data Science. O'Reilly, 2017.] (https://r4ds.had.co.nz/index.html)
```

Comment:
Awesome! For links, don't leave a gap between the square brackets and the parenthesis, otherwise markdown doesn't render the link properly. (You can see that the correct links are a different colour)

Reply:
I have removed the gaps in the document and commited.  Not sure what I do now @dblana

Reply:
That's great. Now you should click the "Ready for Review" button below. At the moment this pull request is "draft", so it cannot be merged. (In theory we should have waited to review after you had flagged it as "ready to review", but it makes no difference in practice 😄)